R/infer_entity_type.R

```diff
@@ -20,7 +20,6 @@
 #' res2 <- infer_entity_type(entity_id=entity_id2, entity_list=NULL)
 #' }
 #' @export
-#' @keywords internal
 
 infer_entity_type <- function(entity_id, entity_list=NULL) {
```

Comment:
@AK197624 This function calls the `.list_api_contents` function, which is defined in a separate file. In the roxygen2 documentation for `infer_entity_type`, you want to have a line that looks like this:
`#' @include .list_api_contents.R`
The `@include` directive will have the effect of making the `.list_api_contents` function available to `infer_entity_type`. Then, running `devtools::document` will modify the `DESCRIPTION` file to make this happen via the `Collate` field. You'll want to do this for every function that needs to access another function that is defined in a separate file in this package.